### DISCUSSION TITLE
In the ngrinder cluster mode, tasks in different regions cannot be executed in paralle

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#847] [Q&A] [answered]
the task still need to wait,Where can I set this？
@junoyoon 
<img width="1011" alt="c732c26d7134ef66f19c6b1d6957815" src="https://user-images.githubusercontent.com/50607317/141881128-e1e9cf1a-762c-4a13-ac98-9030152042ca.png">

**Comment by donggyu04** [comment, marked as answer]
@zhougy1989 

Users can run only one test at the same time, regardless of region.

**Reply by donggyu04** [reply]
@zhougy1989 

This is to prevent monopolizing the use of agents.

**Reply by zhougy1989** [reply]
OK，i know,thks

**Reply by zhougy1989** [reply]
sorry,i have a new question。if this only support one test，What is the significance and role of clusters？
Isn't it more convenient to deploy multiple sets of single controller and multi-agent machines？
@donggyu04

**Reply by donggyu04** [reply]
@zhougy1989 

Clusters exist to manage agents by region and make them available through a single user interface.
Regions can usually be mapped to your IDC.
